views/update_project.erb

```diff
@@ -26,6 +26,10 @@
       <label for="webhookURL">Webhook URL</label>
       <input type="text" name="webhook_url" class="form-control"  id="webhookURL" value="<%= @project.webhook_url %>">
     </div>
+    <div class="form-group">
+      <label for="inputAccessToken">Access Token</label>
+      <input type="password" name="access_token" class="form-control"  id="inputAccessToken" value="<%= @project.access_token %>">
```

Comment:
`value="<%= @project.access_token %>"`

hmm...accessToken is exposed on source html.